hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheService.java

```diff
@@ -103,6 +106,7 @@ public final void init(NodeEngine nodeEngine, Properties properties) {
         }
         this.cacheEventHandler = new CacheEventHandler(nodeEngine);
         this.cacheSplitBrainHandler = new CacheSplitBrainHandler(nodeEngine, configs, segments);
+        this.cacheWanEventPublisher = new CacheWanEventPublisherImpl();
```

Comment:
Yes, `CacheWanEventPublisherImpl` has no purpose here. I'll delete it, and throw `UnsupportedOperationException`.